test/verify/check-multi-machine

```diff
@@ -109,7 +109,7 @@ def change_ssh_port(machine, address, port=None, timeout_sec=120):
     error = None
     while (time.time() - start_time) < timeout_sec:
         try:
-            machine.execute("true", quiet=True)
+            machine.execute("ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -o CheckHostIP=no -o PasswordAuthentication=no -p {0} {1} 2>&1 | grep -q 'Permission denied'".format(port, address), quiet=True)
             return
```

Comment:
The address must be correct, otherwise you'd get a connection error instead of "permission denied". All of our VMs should share the same identity file.

Reply:
Sorry, was wrong on my being wrong; it's not reachable. So we'd have to copy the identity file into the first vm to ssh to the third vm.

Reply:
Ack, let's keep the grep, otherwise we'd have to copy the identity file, and we want to test adding a machine with ssh password.